Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -354,5 +366,55 @@ private void OnCommandBarChanged(string panelName, CommandBar commandbar)
                 panel.Children.Add(commandbar);
             }
         }
+
+        /// <summary>
+        /// Sets whether the selected item should change when focused with the keyboard based on the view state
+        /// </summary>
+        /// <param name="viewState">the view state</param>
+        private void SetListSelectionWithKeyboardFocusOnVisualStateChanged(MasterDetailsViewState viewState)
```

Comment:
This doesn't get called when the initial state is narrow view so when the control is loaded in narrow view, the focus triggers selection and won't work until the control is resized.

Reply:
Please read my last comment on the conversation.